assets/i18n/ar.json

```diff
@@ -1,208 +1,207 @@
-    "I18N_SIGNUP_CLOSE_BUTTON": "أغلق",
-    "I18N_SIGNUP_COMPLETE_REGISTRATION": "إتمام تسجيلك",
-    "I18N_SIGNUP_DO_NOT_SEND_EMAILS": "لا ترسل هذه الرسائل الإلكترونية",
-    "I18N_SIGNUP_EMAIL": "البريد الإلكتروني",
-    "I18N_SIGNUP_EMAIL_PREFERENCES": "تفضيلات البريد الإلكتروني",
-    "I18N_SIGNUP_EMAIL_PREFERENCES_EXPLAIN": "يمكنك تغيير هذا الإعداد في أي وقت من صفحة تفضيلات الخاصة بك.",
-    "I18N_SIGNUP_ERROR_MUST_AGREE_TO_TERMS": "لتعديل الاستكشافات في هذا الموقع; ستحتاج إلى توافق على شروط الموقع.",
-    "I18N_SIGNUP_ERROR_NO_USERNAME": "الرجاء إدخال اسم مستخدم.",
-    "I18N_SIGNUP_ERROR_USERNAME_MORE_50_CHARS": "اسم مستخدم يمكن أن يتكون من 50 حرفا كحد أقصى.",
-    "I18N_SIGNUP_ERROR_USERNAME_NOT_AVAILABLE": "اسم المستخدم هذا غير متوفر.",
-    "I18N_SIGNUP_ERROR_USERNAME_ONLY_ALPHANUM": "يمكن لأسماء المستخدمين أن تتكون من أحرف أبجدية فقط.",
-    "I18N_SIGNUP_ERROR_USERNAME_TAKEN": "عذرا، اسم المستخدم هذا مسجل بالفعل.",
-    "I18N_SIGNUP_ERROR_USERNAME_WITH_ADMIN": "أسماء المستخدمين التي بها 'admin' محجوزة.",
-    "I18N_SIGNUP_ERROR_USERNAME_WITH_SPACES": "يُرجَى التأكد من اسم المستخدم الخاص بك يحتوي على مسافات.",
-    "I18N_SIGNUP_FIELD_REQUIRED": "هذا الحقل مطلوب.",
-    "I18N_SIGNUP_LOADING": "جارٍ التحميل...",
-    "I18N_SIGNUP_PAGE_TITLE": "تاريخ المجتمع - أوبيا",
-    "I18N_SIGNUP_SEND_ME_NEWS": "أرسل لي الأخبار والتحديثات حول الموقع",
-    "I18N_SIGNUP_SITE_DESCRIPTION": "<[sitename]> هو مشاع مفتوح من مصادر التعلم. جميع المواد  قابلة لإعادة الاستخدام والمشاركة بحرية.",
-    "I18N_SIGNUP_SITE_OBJECTIVE": "<[sitename]> موجود لتشجيع الإبداع والتحسين المستمر لمجموعة من المصادر التعليمية عالية الجودة التي هي متاحة بحرية لأي شخص.",
-    "I18N_SIGNUP_UPDATE_WARNING": "يُرجَى ملاحظة أننا قد تم تحديث شروط الاستخدام الخاصة بنا مؤخرا.",
-    "I18N_SIGNUP_USERNAME_EXPLANATION": "سيتم عرض اسم المستخدم الخاص بك مقابل مساهماتك.",
-    "I18N_SIGNUP_WHY_LICENSE": "لماذا CC-BY-SA؟",
-    "I18N_SPLASH_JAVASCRIPT_ERROR_THANKS": "شكرا.",
-    "I18N_SPLASH_JAVASCRIPT_ERROR_TITLE": "نحن بحاجة إلى تفعيل جافا سكريبت في متصفحك",
-    "I18N_SPLASH_PAGE_TITLE": "أوبيا:تعليم، تعلم، استكشاف",
-    "I18N_SPLASH_SECOND_EXPLORATION_DESCRIPTION": "الاستكشافات بسيطة للإنشاء، وتحديثها بسهولة مبني على التغذية المرتدة الفردية للطالب والاتجاهات في تجربة المتعلمين في جميع أنحاء العالم.",
-    "I18N_SPLASH_SITE_FEEDBACK": "أرسل تعليقا",
-    "I18N_SPLASH_TITLE": "التفكير خارج الكتب.",
-    "I18N_TOPNAV_ABOUT_OPPIA": "عن أوبيا",
-    "I18N_TOPNAV_BLOG": "مدونة",
-    "I18N_TOPNAV_CONTACT_US": "اتصل بنا",
-    "I18N_TOPNAV_DASHBOARD": "لوحة المنشئ",
-    "I18N_TOPNAV_FORUM": "المنتدى",
-    "I18N_TOPNAV_GET_STARTED": "ابدأ",
-    "I18N_TOPNAV_LIBRARY": "المكتبة",
-    "I18N_TOPNAV_LOGOUT": "خروج",
-    "I18N_TOPNAV_NOTIFICATIONS": "الإشعارات",
-    "I18N_TOPNAV_PREFERENCES": "التفضيلات",
-    "I18N_TOPNAV_SIGN_IN": "تسجيل الدخول",
-    "I18N_TOPNAV_TEACH_WITH_OPPIA": "التعليم مع أوبيا"
+	"I18N_ABOUT_PAGE_TITLE": "من نحن - أوبيا",
```

Comment:
Translatewiki uses tabs. I don't want to convert them to spaces each time we do a merge, and we've newly introduced functionality to ignore the assets/i18n files when linting. So let's default to tabs for files in this directory, since they aren't meant to be human-edited anyway.